dspace-server-webapp/src/test/java/org/dspace/app/rest/WorkspaceItemRestRepositoryIT.java

```diff
@@ -347,25 +491,110 @@ public void deleteOneTest() throws Exception {
+                .build();
+        WorkspaceItem witem = WorkspaceItemBuilder.createWorkspaceItem(context, col1)
+                .withTitle("Workspace Item 1")
+                .withIssueDate("2019-01-01")
+                .build();
+
+        Item item = witem.getItem();
+
+        //Add a bitstream to the item
+        String bitstreamContent = "ThisIsSomeDummyText";
+        Bitstream bitstream = null;
+        try (InputStream is = IOUtils.toInputStream(bitstreamContent, CharEncoding.UTF_8)) {
+            bitstream = BitstreamBuilder
+                    .createBitstream(context, item, is)
+                    .withName("Bitstream1")
+                    .withMimeType("text/plain").build();
+        }
+
+        context.restoreAuthSystemState();
+
+        getClient().perform(delete("/api/submission/workspaceitems/" + witem.getID()))
+                   .andExpect(status().isUnauthorized());
+    }
+
+    @Test
+    public void deleteOneForbiddenTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        EPerson submitter1 = EPersonBuilder.createEPerson(context)
+                .withEmail("submitter1@example.com")
+                .withPassword("qwerty01")
+                .build();
+        EPerson submitter2 = EPersonBuilder.createEPerson(context)
+                .withEmail("submitter2@example.com")
+                .withPassword("qwerty02")
+                .build();
+
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                .withName("Parent Community")
+                .build();
+        Collection col1 = CollectionBuilder.createCollection(context, parentCommunity)
+                .withName("Collection 1")
+                .withSubmitterGroup(submitter1)
+                .build();
+
+        context.setCurrentUser(submitter1);
+        WorkspaceItem witem = WorkspaceItemBuilder.createWorkspaceItem(context, col1)
+                .withTitle("Workspace Item 1")
+                .withIssueDate("2020-01-01")
+                .build();
+
+        Item item = witem.getItem();
+
+        //Add a bitstream to the item
+        String bitstreamContent = "ThisIsSomeDummyText";
+        Bitstream bitstream = null;
+        try (InputStream is = IOUtils.toInputStream(bitstreamContent, CharEncoding.UTF_8)) {
+            bitstream = BitstreamBuilder
+                    .createBitstream(context, item, is)
+                    .withName("Bitstream1")
+                    .withMimeType("text/plain").build();
+        }
+
+        context.restoreAuthSystemState();
+
+        String tokenSubmitter2 = getAuthToken(submitter2.getEmail(), "qwerty02");
+        getClient(tokenSubmitter2).perform(delete("/api/submission/workspaceitems/" + witem.getID()))
+                                  .andExpect(status().isForbidden());
+
+        String authToken = getAuthToken(admin.getEmail(), password);
+        getClient(authToken).perform(get("/api/submission/workspaceitems/" + witem.getID()))
```

Comment:
I think this should be a `delete()` request, not a `get()`, as you are testing deletion in this method.

Also, we should also be verifying that a Submitter (in this case `submitter1`) can delete their own WorkspaceItems.

Reply:
it is intentional, the delete operation is checked two rows above. This get is an extra security check to verify that regardless to the status returned to the client the server has really NOT deleted the workspaceitem

Reply:
@abollini : Oh, you are right, I misread this code.  the `get()` is correct here.

I also see now that I overlooked that we already have a test to verify that a submitter can delete their own WorkspaceItem (`deleteOneTest()`).  So, I'll resolve this conversation/comment.